packages/rnv/src/configTools/configParser.js

```diff
@@ -271,7 +271,8 @@ export const versionCheck = async (c) => {
                     c.runtime.rnvVersionRunner
                 )} against project built with rnv v${chalk.red(
                     c.runtime.rnvVersionProject
-                )}. This might result in unexpected behaviour! It is recommended that you run your rnv command with npx prefix: ${recCmd} . or manually update your devDependencies.rnv version in your package.json.`
+                )}. This might result in unexpected behaviour! It is recommended that you run your rnv command with npx prefix: ${recCmd} . or manually update your devDependencies.rnv version in your package.json.`,
+                default: actionNoUpdate
```

Comment:
Wasn't too sure about which default here. Can change if required

Reply:
this fails on purpose. under CI (--ci flag) environment your rnv should never be different to avoid unexpected results.
I'll update this as a part of review commits